### PR TITLE
Retry installing chrony

### PR DESCRIPTION
**Pull request by xuto2**

### CODE REVIEW

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 1]
```diff
@@ -89,15 +89,15 @@ updateAptWithMicrosoftPkg() {
 
 disableNtpAndTimesyncdInstallChrony() {
     # Disable systemd-timesyncd
-    sudo systemctl stop systemd-timesyncd
-    sudo systemctl disable systemd-timesyncd
+    systemctl_stop 20 30 120 systemd-timesyncd || exit $ERR_STOP_SYSTEMD_TIMESYNCD_TIMEOUT
```
Comment (bowang-666) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:92]: actually there's a func systemctlDisableAndStop that could do these two steps together

Comment (xuto2) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:92]: checking.

Comment (xuto2) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:92]: systemctlDisableAndStop  doesn't error out. It needs more change

**Review by bowang-666 — APPROVED** [review]

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 2]
```diff
@@ -89,15 +89,15 @@ updateAptWithMicrosoftPkg() {
...
+    systemctl disable systemd-timesyncd
```
Comment (bowang-666) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:93]: systemctl_disable systemd-timesyncd || exit err

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh [hunk 3]
```diff
@@ -89,15 +89,15 @@ updateAptWithMicrosoftPkg() {
...
     # Disable ntp
-    sudo systemctl stop ntp
-    sudo systemctl disable ntp
+    systemctl_stop 20 30 120 ntp || exit $ERR_STOP_NTP_TIMEOUT
+    systemctl disable ntp
```
Comment (bowang-666) [inline parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:96]: systemctl_disable 20 30 120 ntp || exit err

File: parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh (hunk 1 above)
Comment (bowang-666) [reply parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh:92]: nvm on this

**Pull request merged**